inc/functions/admin.php

```diff
@@ -76,21 +76,23 @@ function rocket_renew_box( $function, $uid = 0 ) {
 }
 
 /**
- * Dismissed 1 box, wrapper of rocket_dismiss_boxes()
+ * Dismiss one box.
  *
  * @since 1.3.0
+ * @since 3.6 Doesn’t die anymore.
  *
- * @param string $function function name.
- * @return void
+ * @param string $function Function (box) name.
  */
 function rocket_dismiss_box( $function ) {
-	rocket_dismiss_boxes(
-		[
-			'box'      => $function,
-			'_wpnonce' => wp_create_nonce( 'rocket_ignore_' . $function ),
-			'action'   => 'rocket_ignore',
-		]
-	);
+	global $current_user;
```

Comment:
Instead of the global, let's use `wp_get_current_user()`.

Reply:
https://github.com/wp-media/wp-rocket/pull/2616/commits/09ba8a9c695a06ab5525478c83a5853a56e290a3